bin/run_brick.rb

```diff
@@ -14,28 +14,65 @@ def get_brick_params(prefix)
   ENV.select { |k,| k.to_s.match(/^#{prefix}.*/) }.map { |k, v| [k.slice(prefix.length..-1), v] }.to_h
 end
 
+def log_execution_summary_error(params, log, brick_type, error_message)
+  result = {
+    errorCode: 'lcm.error',
+    message: error_message,
+    parameters: []
+  }
+
```

Comment:
1. JSON should have root element executionResult, status=OK or ERROR. In the future we can extend it for WARN status.
{
  "executionResult": {
    "status": "OK", // ERROR
    "message": "Blabla",
  }
}

2. log_execution_summary_error --> log_execution_result(status, message)

Reply:
3. Define enum for status OK, ERROR